xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1263,19 +1263,15 @@ class FlashAttentionBMMScaleSlidingWindowMaskSoftmaxBMM
   }
 };
 
-class FlashAttentionBMMScalePaddingMaskSoftmaxBMMF8
+class FlashAttentionBMMScaleSoftmaxBMMF8
     : public MultiHeadedAttentionTest {
  protected:
-  void TestImpl_Flash_Attention_Inference_BMM1_NoMask_Softmax_BMM2_F8() {
-    if (skip_reason_) GTEST_SKIP() << *skip_reason_;
-    if (GetDnnVersionInfoOrDefault(backend().default_stream_executor()) <
-        se::dnn::VersionInfo(9, 1, 0)) {
-      GTEST_SKIP() << "Flash Attention requires cuDNN >= 9.1.0.";
-    }
-    XlaBuilder builder(TestName());
-    std::string hlo_string_ref =
+  const std::string                                       // NOLINT
```

Comment:
Why is the nolint needed here? What's the error message? Is it that `const` here is not needed? It really isn't, so please remove it. In fact, it's likely better if you make this function return an absl::string_view and change the type inside the function to `static constexpr absl::string_view = {".."};`

Reply:
Thanks for the suggestion. The NOLINTs are removed.